tests/Unit/Evolution/DiscontinuousGalerkin/Initialization/Test_QuadratureTag.cpp

```diff
@@ -0,0 +1,23 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <string>
+
+#include "Evolution/DiscontinuousGalerkin/Initialization/QuadratureTag.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "NumericalAlgorithms/Spectral/Spectral.hpp"
+
+SPECTRE_TEST_CASE("Unit.Evolution.DG.Initialization.QuadratureTag",
+                  "[Unit][Evolution]") {
+  TestHelpers::db::test_simple_tag<evolution::dg::Tags::InitialQuadrature>(
+      "InitialQuadrature");
+  CHECK(TestHelpers::test_creation<Spectral::Quadrature,
+                                   evolution::dg::OptionTags::Quadrature>(
+            "Gauss") == Spectral::Quadrature::Gauss);
+  CHECK(TestHelpers::test_creation<Spectral::Quadrature,
+                                   evolution::dg::OptionTags::Quadrature>(
+            "GaussLobatto") == Spectral::Quadrature::GaussLobatto);
```

Comment:
This is like the test of the yaml creation that I expected, but for the tag? What are our "best practices" here? Should both be tested using nearly-identical code?

Reply:
I think both should be tested. The tag should work, but the yaml creation should work with without the tag :)